boskos/janitor/janitor.go

```diff
@@ -61,11 +68,11 @@ func setup(c boskosClient, janitorCount int, bufferSize int) chan string {
 	return buffer
 }
 
-func run(c boskosClient, buffer chan string) int {
+func run(c boskosClient, buffer chan string, res string) int {
 	totalAcquire := 0
 
 	for {
-		if proj, err := c.Acquire("project", "dirty", "cleaning"); err != nil {
+		if proj, err := c.Acquire(res, "dirty", "cleaning"); err != nil {
```

Comment:
AKA my question is not about the value of making it easy for the janitor to handle multiple types of projects w/o changing the code (makes sense).

My question is about why we want to create, manage and update a deployment for every type of project? Rather than having a single deployment handle multiple project types (driven by config)